### PR TITLE
XeroArch Ascii logo

### PR DESCRIPTION
**Pull request by XeroArch**
Our ASCII logo to use in our upcoming ArchLinux distribution: XeroArch

### CODE REVIEW

**Comment by CarterLi** [comment]
An ascii logo must be smaller than 50x20 (in number of charactors)

**Comment by XeroArch** [comment]
> An ascii logo must be smaller than 50x20 (in number of charactors)

All right, but i checked some of the ASCII logos available there in the ascii folder and some of them have even more than 1100 characters

**Comment by CarterLi** [comment]
Let's get rid of them

**Comment by XeroArch** [comment]
> Let's get rid of them

updated the ascii logo, now it has 1171 characters, but it is because of the fact that i needed to to specify the correct colors that it use on its parts (${c(number of the color)}).

Originally the logo itself without colors is 778 characters.

File: src/logo/ascii/ascii-art.txt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          @%%%%%%%==============================%%%%=:::::::::::::::::::::::::::::+%%%%%%@          
+         @%%%%%%%===============================%%%%=::::::::::::::::::::::::::::::+%%%%%%@         
+        @%%%%%%#================================%%%%%-::::::::::::::::::::::::::::::=%%%%%%@        
+       @%%%%%%#=================================*%%%%#-::::::::::::::::::::::::::::::=%%%%%%@       
+      @%%%%%%#==================================+%%%%%%=::::::::::::::::::::::::::::::=%%%%%%@      
+     @%%%%%%#====================================#%%%%%%%*-::::::::::::::::::::::::::::=%%%%%%@     
+    @%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%@    
+     @%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%@     
+      @@%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%@@      
+
```
Comment (CarterLi) [inline src/logo/ascii/ascii-art.txt:45]: Remove this file

File: src/logo/ascii/xeroarch.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+${c1}
```
Comment (CarterLi) [inline src/logo/ascii/xeroarch.txt:1]: Remove this line

File: src/logo/ascii/xeroarch.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+${c1}     ███${c6}▓▓▓▓▓${c1}████${c3}▓${c4}▒▒▒▒${c3}▓${c1}▓███${c6}▓▓▓▓▓${c1}███
+${c1}    ███${c6}▓▓▓▓▓▓▓▓▓${c1}███▓▓███${c6}▓▓▓▓▓▓▓▓▓${c1}███
+${c1}   ███${c6}▓▓▓▓▓▓▓${c2}▓▓${c6}▓▓▓▓▓${c1}█${c6}▓▓▓▓▓▓▓▓▓▓▓▓▓${c1}███
+${c1}  ███${c6}▓▓▓▓▓▓${c2}▓▓${c5}▒▒${c6}${c2}▓▓${c6}▓▓▓${c1}▓${c6}▓▓▓▓${c4}()${c6}▓▓▓▓▓▓▓▓${c1}███
+${c1} ███${c6}▓▓▓▓▓${c2}▓▓${c5}▒▒▒▒▒▒${c6}${c2}▓▓${c6}▓${c1}▓${c6}▓▓${c3}()${c6}▓▓${c3}()${c6}▓▓▓▓▓▓▓${c1}███
+${c1}███${c6}▓▓▓▓▓▓▓▓${c2}▓▓${c5}▒▒${c6}${c2}▓▓${c6}▓▓▓${c1}▓${c6}▓▓▓▓${c4}()${c6}▓▓▓▓▓▓▓▓▓▓${c1}███
+${c1}██${c6}▓▓▓▓▓▓▓▓▓▓▓${c2}▓▓${c6}▓▓▓▓▓${c1}▓${c6}▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓${c1}██
+${c1}████████████████████████████████████████
+
+
```
Comment (CarterLi) [inline src/logo/ascii/xeroarch.txt:22]: Remove trailing newlines

**Pull request merged**